src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -198,14 +198,15 @@ private enum ResponseStatus
             return authParams;
         }
 
-        private static MsalException DecorateExceptionWithRuntimeErrorProperties(MsalException exception, Error runtimeError)
-        {
+        private static MsalException DecorateExceptionWithRuntimeErrorProperties(MsalException exception, AuthResult wamAuthResult)
+        {            
             var result = new Dictionary<string, string>()
             {
-                { MsalException.BrokerErrorContext, runtimeError.Context },
-                { MsalException.BrokerErrorTag, $"0x{runtimeError.Tag:X}" },
-                { MsalException.BrokerErrorStatus, runtimeError.Status.ToString() },
-                { MsalException.BrokerErrorCode, (runtimeError.ErrorCode).ToString() },
+                { MsalException.BrokerErrorContext, wamAuthResult?.Error.Context },
```

Comment:
>wamAuthResult?.Error.Context } [](http://example.com/codeflow?start=52&length=30)

When PII collection is enabled context could contain PII do we need to take that into account ?

Reply:
We have PII flag too, which we inject in msal_rutime, so there's no danger of emitting PII info unless app developer explicitly sets it to true.